packages/softether-vpn/build.sh

```diff
@@ -0,0 +1,44 @@
+TERMUX_PKG_HOMEPAGE=https://www.softether.org/
+TERMUX_PKG_DESCRIPTION="An open-source cross-platform multi-protocol VPN program"
```

Comment:
Is this package used for setting up an softether vpn server, or is it a client program for connecting to a server?
Does it work without root?

Reply:
Contains both client and server. Some part requires root and other parts do not.

Again, the problem is that there is no trivial way to separate client and server builds.

Reply:
@xtkoba As a user, I personally don't mind that. I appreciate it if I could run the SoftEther server as well as the client from the same package